plugins/TodayAtMun/__init__.py

```diff
@@ -0,0 +1,57 @@
+import discord
+from discord.ext import commands
+import typing
+
+from Bot import bot
+from Plugin import AutomataPlugin
+from plugins.TodayAtMun.DiaryParser import DiaryParser
+from plugins.TodayAtMun.Today import Today
+
+
+class TodayAtMun(AutomataPlugin):
+    def __init__(self, manifest, bot: commands.Bot):
+        super().__init__(manifest, bot)
+        self.parse = DiaryParser()
+        self.tod = Today(self.parse.diary)
+
+    @commands.group()
+    async def today(self, ctx: commands.Context):
+        """ Provides brief info of significant dates on the Mun calendar"""
```

Comment:
Few things going on here:
* Unnecessary leading space in docstring
* Incorrect capitalization of "MUN"
* Missing trailing period